robottelo/constants.py

```diff
@@ -1,6 +1,7 @@
 # -*- encoding: utf-8 -*-
 """Defines various constants"""
 from nailgun import entities
+from robottelo.config import settings
```

Comment:
Is circular dependency between settings and constants required, and is it really good idea?

As a rule of thumb, constants are values that require rebuilding and redeploying application. Settings are values provided at run time, and possibly changed at run time.

Values that remain constant during application life, but depend on settings (like the ones you have below), should be handled by dynamically constructed map or be provided by helper function that knows how to create them.

I would recommend new module, which may live in `constants` namespace. So import order will be:

1. constants don't have any import requirements
2. settings imports constants
3. constants.repos imports settings <- whenever that happens, settings are already configured

In Python, you can also add new properties dynamically, so technically settings module could construct these constants and add them back to constants module. That should allow you to solve circular dependency without users knowing there was a change, but will be surprising to anyone who decides to take a look at how things actually work.

Reply:
Even I was thinking to create the different `constants` package in robottelo and hence deciding tones of constants in separate modules. But not sure if this is the right PR to do that :(

Reply:
If I do that here then lot of test modules have to consume/update this change :)